### PR TITLE
dispatcher: rework attrs parameter in rpc 'add' command

### PR DESCRIPTION
**Pull request by grumvalski**
#### Pre-Submission Checklist

- [x] Commit message has the format required by CONTRIBUTING guide
- [x] Commits are split per component (core, individual modules, libs, utils, ...)
- [x] Each component has a single commit (if not, squash them into one commit)
- [x] No commits to README files for modules (changes must be done to docbook files
in `doc/` subfolder, the README file is autogenerated)

#### Type Of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:

- [ ] PR should be backported to stable branches
- [x] Tested changes locally
- [ ] Related to issue #XXXX (replace XXXX with an open issue number)

#### Description
Fixed attrs parameters handling i rpc 'add' command.

### CODE REVIEW

**Comment by henningw** [comment]
related to review remarks from #2336

File: src/modules/dispatcher/dispatcher.c [hunk 1]
```diff
@@ -1798,15 +1798,19 @@ static const char *dispatcher_rpc_add_doc[2] = {
...
+	str attrs;
 
 	flags = 0;
 
-	if(rpc->scan(ctx, "dS*d", &group, &dest, &flags, &attrs) < 3) {
+	nparams = rpc->scan(ctx, "dS*dS", &group, &dest, &flags, &attrs);
+	if(nparams < 2) {
 		rpc->fault(ctx, 500, "Invalid Parameters");
 		return;
+	} else if (nparams < 4) {
```
Comment (miconda) [inline src/modules/dispatcher/dispatcher.c:1811]: I think the condition here has to be `(nparams < 4)`.

Comment (grumvalski) [reply src/modules/dispatcher/dispatcher.c:1811]: right, fixed and force pushed.

**Comment by miconda** [comment]
Thanks, can be merged.

**Comment by henningw** [comment]
LGTM analysis failed due to out of memory - but I guess this is not related to this commit.

**Pull request merged**